emmet-core/emmet/core/molecules/vibration.py

```diff
@@ -18,95 +18,36 @@
 __author__ = "Evan Spotte-Smith <ewcspottesmith@lbl.gov>"
 
 
-class SpectrumDoc(MoleculeMetadata):
-    """
-    Base model definition for any spectra document. This should contain
-    metadata on the structure the spectra pertains to
-    """
+class VibrationDoc(PropertyDoc):
 
-    spectrum_name: str
+    property_name = "vibrations"
 
-    molecule_id: MPID = Field(
+    molecule: Molecule = Field(
         ...,
-        description="The ID of the molecule, used as a universal reference across property documents."
+        description="Molecular structure"
     )
 
-    spectrum_id: str = Field(
+    frequencies: List[float] = Field(
         ...,
-        title="Spectrum Document ID",
-        description="The unique ID for this spectrum document",
+        description="List of molecular vibrational frequencies"
     )
 
-    last_updated: datetime = Field(
-        description="Timestamp for the most recent calculation update for this property",
-        default_factory=datetime.utcnow,
+    frequency_modes: List[List[List[float]]] = Field(
+        ...,
+        description="Vibrational frequency modes of the molecule"
```

Comment:
(units: 1/cm) ?

Reply:
The frequencies are in inverse centimeters. Not the modes.